src/lib/libhtml5.js

```diff
@@ -249,6 +249,7 @@ var LibraryHTML5 = {
     },
   },
 
+  $registerKeyEventCallback__noleakcheck: true,
```

Comment:
Is this intentional in these functions? Are they not allocating events the user must deallocate?

Reply:
They are all allocating the event struct on first use an never free'ing it.